### DISCUSSION TITLE
External perimeters first for x layers

### DISCUSSION THREAD

**Discussion by matzydude** [supermerill/SuperSlicer#2981] [General]
Is there (or can there be) an option to print the external perimeters first for a set amount of layers, for example for the first 2 layers only? I have a print that has some really small "blobs" of print laid down in the first layer and often come away, however, if I print external perimeters first it works perfectly, but later in the print some overhangs fail due to the external perimeters being printed first !!

**Comment by supermerill** [comment]
* right-click on the object (or right panel) -> layer hieght modifier.
* set the z and height of the modifer
* click on the cog of the modifier -> add settings -> Periemter shells
* select the "external periemter first" -> apply
* check the settings in the right panel

**Reply by matzydude** [reply]
Thats perfect, thank you very much. I never realised I could do that. Superslicer never fails to impress me.
